data/src/entity_manager/system.py

```
@@ -6104,7 +6130,8 @@ def _unpack_result_e(self, entity_class, field_names, options, result_set):
                         # list and in case it's not sets it in the relations list,
                         # this is an expensive operation (presence check in list)
                         if not _new_entity in relation_list:
-                            _new_entity and relation_list._append(_new_entity)
+                            if _new_entity:
+                                relation_list._append(_new_entity)
```

Pylint (3.3.7)

[warning] Access to a protected member _append of a client class

(W0212)